azure-pipelines.yml

```diff
@@ -59,6 +65,106 @@ jobs:
       projects: '**/*UnitTest/*.csproj'
       arguments: '--configuration $(buildConfiguration)'
 
+  - task: DotNetCoreCLI@2
+    displayName: 'E2E tests for Spark 2.3.0 with oldest compatible worker $(oldestCompatibleWorker)'
+    inputs:
+      command: test
+      projects: '**/Microsoft.Spark.E2ETest/*.csproj'
+      arguments: '--configuration $(buildConfiguration) --filter (FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.DataFrameTests.TestDataFrameGroupedMapUdf)&(FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.DataFrameTests.TestDataFrameVectorUdf)&(FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.BroadcastTests.TestDestroy)&(FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.BroadcastTests.TestMultipleBroadcastWithoutEncryption)&(FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.BroadcastTests.TestUnpersist)&(FullyQualifiedName!=Microsoft.Spark.E2ETest.UdfTests.UdfSimpleTypesTests.TestUdfWithReturnAsTimestampType)&(FullyQualifiedName!=Microsoft.Spark.E2ETest.UdfTests.UdfSimpleTypesTests.TestUdfWithTimestampType)'
```

Comment:
Can we use variables for these please?

Reply:
Yes once I have figured out what all tests need to be filtered, will create variables for them. I am still trying to isolate the reason for tests hanging, looks like after removing timestamp tests it passed 2.3.0 but then hung at 2.3.1. Currently investigating that.

Reply:
It's hanging b/c you filtered only for 2.3.0?

Reply:
Yes, forgot I did that. Ok, so we need to refactor timestamp test in CreateDataFrame in that case. Should I create a separate PR for that or just do it here?

Reply:
Let's do it here so that it's easier to track. Thanks!

Reply:
Sounds good, done that, testing now.